content/en/news/releases/1.11.x/announcing-1.11/change-notes/index.md

```diff
@@ -0,0 +1,256 @@
+this automatically.
+  
+
+- **Added** a flag setting ShutdownDuration in pilot and its default value is 10s.
+  
+
+- **Added** an environment variable `PILOT_STATUS_UPDATE_INTERVAL` that is the interval to update the XDS distribution status and its default value is `500ms`.
+  
+
+- **Added** The Istio sidecar agent will offer HTTP endpoint localhost:15009/debug/<typeurl>.  GET requests
+to that URL will be resolved by sending an xDS discovery "event" to istiod.  This can be disabled by setting
+the following in the Istio Operator: meshConfig.defaultConfig.proxyMetadata.PROXY_XDS_DEBUG_VIA_AGENT=false
+  ([Issue #22274](https://github.com/istio/istio/issues/22274))
+
+- **Added** support for overriding the locality of the WorkloadGroup template in
+auto registered WorkloadEntries. Locality overrides can be passed in through
+Envoy bootstrap configuration.
+  ([Issue #33426](https://github.com/istio/istio/pull/33426)),([Issue #33426](https://github.com/istio/istio/issues/33426))
+
+- **Added** new metric for tracking distribution of configuration resource sizes being pushed by istiod.
+  ([Issue #31772](https://github.com/istio/istio/issues/31772))
+
+- **Added** experimental support for the Kubernetes Multi-Cluster Services (MCS) host (`clusterset.local`).
+This feature is off by default, but can be enabled by setting the following flags in Istio:
+`ENABLE_MCS_HOST` and `ENABLE_MCS_SERVICE_DISCOVERY`. When enabled Istio will include the MCS host as a
+domain in the service's HTTP route. Additionally, Istio will support the MCS host during a DNS lookup.
+For now, the MCS host is just an alias for `cluster.local` and resolves to the same service IP.
+Future work will give the MCS host a separate IP as is defined by the MCS spec.  ([Issue #33949](https://github.com/istio/istio/issues/33949))
+
+- **Added** experimental support for controlling service endpoint discoverability with Kubernetes Multi-Cluster
+Services (MCS). This feature is off by default, but can be enabled by setting the
+`ENABLE_MCS_SERVICE_DISCOVERY` flag in Istio. When enabled, Istio will make service endpoints
+only discoverable from within the same cluster by default. To make the service endpoints within a cluster
+discoverable throughout the mesh, a `ServiceExport` CR must be created within the same cluster as the service
+endpoints. this process can be automated by enabling the Istio flag `ENABLE_MCS_AUTOEXPORT`. With this enabled,
+Istio will automatically create `ServiceExport` in all clusters for each service.
+  ([Issue #29384](https://github.com/istio/istio/issues/29384))
+
+- **Added** Specify `meshConfig.discoverySelectors` to dynamically restrict the set of namespaces for Services, Pods, and Endpoints that istiod processes when pushing xDS updates to improve performance on the data plane.
+  ([Issue #26679](https://github.com/istio/istio/issues/26679))
+
```

Comment:
Should we remove this?  Pretty sure this was in 1.10's release also.  https://preliminary.istio.io/latest/news/releases/1.10.x/announcing-1.10/change-notes/#traffic-management

Reply:
Probably. Guessing the release note file made it into 1.11 but not 1.10?